src/main/java/org/gridsuite/network/map/NetworkMapController.java

```diff
@@ -309,36 +232,12 @@ public class NetworkMapController {
-    public @ResponseBody List<SubstationMapData> getMapSubstationsData(@Parameter(description = "Network UUID") @PathVariable("networkUuid") UUID networkUuid,
-                                           @Parameter(description = "Variant Id") @RequestParam(name = "variantId", required = false) String variantId,
-                                           @Parameter(description = "Substations id") @RequestParam(name = "substationId", required = false) List<String> substationsIds) {
-        return networkMapService.getMapSubstations(networkUuid, variantId, substationsIds);
-    }
-
-    @GetMapping(value = "/networks/{networkUuid}/map-lines", produces = APPLICATION_JSON_VALUE)
-    @Operation(summary = "Get lines description")
-    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "lines description to render network map")})
-    public @ResponseBody List<LineMapData> getMapEquipmentsData(@Parameter(description = "Network UUID") @PathVariable("networkUuid") UUID networkUuid,
-                                                                @Parameter(description = "Variant Id") @RequestParam(name = "variantId", required = false) String variantId,
-                                                                @Parameter(description = "Substations id") @RequestParam(name = "substationId", required = false) List<String> substationsIds) {
-        return networkMapService.getMapLines(networkUuid, variantId, substationsIds);
-    }
-
-    @GetMapping(value = "/networks/{networkUuid}/map-hvdc-lines", produces = APPLICATION_JSON_VALUE)
-    @Operation(summary = "Get hvdc lines description")
-    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "lines description to render network map")})
-    public @ResponseBody List<HvdcLineMapData> getHvdcLineMapData(@Parameter(description = "Network UUID") @PathVariable("networkUuid") UUID networkUuid,
-                                                                @Parameter(description = "Variant Id") @RequestParam(name = "variantId", required = false) String variantId,
-                                                                @Parameter(description = "Substations id") @RequestParam(name = "substationId", required = false) List<String> substationsIds) {
-        return networkMapService.getMapHvdcLines(networkUuid, variantId, substationsIds);
-    }
-
     @GetMapping(value = "/networks/{networkUuid}/branch-or-3wt/{equipmentId}", produces = APPLICATION_JSON_VALUE)
     @Operation(summary = "From an equipment ID, get the associated line or 2WT or 3WT")
-    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "Line or 2WT or 3WT description")})
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Line or 2WT or 3WT description"),
+            @ApiResponse(responseCode = "204", description = "No element found")
```

Comment:
I think the appropriate code is 404, not 204, and so it should be managed in the .catch of the front-end.

Reply:
404 is an error response
204 is an optional response

Reply:
answer : decision have been made to use 204